sdk/azidentity/cache/linux_test.go

```diff
@@ -18,9 +18,21 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var ctx = context.Background()
+var (
+	ctx              = context.Background()
+	storageAvailable = false
+)
+
+func TestMain(m *testing.M) {
+	err := tryKeyring()
+	storageAvailable = err == nil
```

Comment:
What kind of error is happening in CI?  I wonder if there's a way we can fix this so we don't have to skip coverage.

Reply:
Turns out there were two issues: a test bug used the wrong path, and the user isn't allowed to search its persistent keyring. I fixed the test bug but the permission issue requires more investigation. I think we should skip the tests in CI for now, to unblock an initial beta release while I figure out whether there's a better way to handle or work around this permission issue.

Reply:
Glad you found the root cause.